### PR TITLE
Host a static website locally using S3 and Terraform with LocalStack

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/tutorials/s3-static-website-terraform/index.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+title: "Host a static website locally using Simple Storage Service (S3) and Terraform with LocalStack"
+linkTitle: "Host a static website locally using Simple Storage Service (S3) and Terraform with LocalStack"
+weight: 4
+description: >
+  Host a static website using a Simple Storage Service (S3) bucket to serve static content by provisioning the infrastructure using Terraform in LocalStack. Learn how to configure S3 buckets locally for testing and integration, and make use of LocalStack's S3 API & `tflocal` CLI to provision infrastructure locally.
+cascade:
+  type: docs
+---
+
+[AWS Simple Storage Service (S3)](https://aws.amazon.com/s3/) is a proprietary object storage solution that can store an unlimited number of objects for many use cases. S3 is a highly scalable, durable and reliable service that we can use for various use cases: hosting a static site, handling big data analytics, managing application logs, storing web assets and much more!
```
Comment (viren-nadkarni) [inline content/en/tutorials/s3-static-website-terraform/index.md:11]: No action necessary here, but it's usually a good idea to have one sentence per line in markdown. This helps with the diffs.

**Review by thrau — APPROVED** [review]
LGTM! would it maybe be useful to put all the terraform code together at the end, so people can copy the entire code?

**Review by steffyP — APPROVED** [review]
Thanks for adding this tutorial @HarshCasper 👍 
I have a few NITs and comments but overall I like it very much 😄

File: content/en/tutorials/s3-static-website-terraform/index.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+In this tutorial, we will deploy a static website using an S3 bucket over a locally emulated AWS infrastructure on LocalStack. We will use Terraform to automate the creation & management of AWS resources by declaring them in the HashiCorp Configuration Language (HCL). We will also learn about `tflocal`, a CLI wrapper created by LocalStack, that allows you to run Terraform locally against LocalStack.
+
+## Prerequisites
+
+For this tutorial, you will need:
+
+- [LocalStack Community](https://github.com/localstack/localstack)
+- [Terraform](https://www.terraform.io/downloads.html)
+- [awslocal](https://github.com/localstack/awscli-local)
+- [tflocal](https://pypi.org/project/terraform-local/)
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply content/en/tutorials/s3-static-website-terraform/index.md:26]: I think removing it entirely would make sense here then 👍

File: content/en/tutorials/s3-static-website-terraform/index.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+$ tflocal init
+$ tflocal plan
+$ tflocal apply
+{{< / command >}}
+
+## Conclusion
+
+In this tutorial, we have seen how to use LocalStack to create an S3 bucket and configure it to serve a static website. We have also seen how you can use Terraform to provision AWS infrastructure in an emulated local environment using LocalStack. You can use [LocalStack Pro](https://app.localstack.cloud) to view the created buckets and files on the LocalStack Resource dashboard for S3 and upload more files or perform other operations on the bucket. Using LocalStack, you can perform various operations using emulated S3 buckets and other AWS services without creating any real AWS resources.
+
+The code for this tutorial can be found in our [LocalStack Terraform samples over GitHub](https://github.com/localstack/localstack-terraform-samples/tree/master/s3-static-website). Further documentation for S3 is available on our [S3 documentation]({{<ref "s3" >}}).
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply content/en/tutorials/s3-static-website-terraform/index.md:336]: I will have it done :)

File: content/en/tutorials/s3-static-website-terraform/index.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+With S3, you have unlimited storage with your data stored in buckets. A bucket refers to a directory, while an object is just another term for a file. Every object (file) stores the name of the file (key), the contents (value), a version ID and the associated metadata. You can also use S3 to host a static website, to serve static content. It might include HTML, CSS, JavaScript, images, and other assets that make up your website.
+
+LocalStack Community supports the S3 API, which means you can use the same API calls to interact with S3 in LocalStack as you would with AWS. Using LocalStack, you can create and manage S3 buckets and objects locally, use AWS SDKs and third-party integrations to work with S3, and test your applications without making any significant alterations. LocalStack also supports the creation of S3 buckets with static website hosting enabled.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply content/en/tutorials/s3-static-website-terraform/index.md:15]: Rephrased to just `LocalStack` instead 😄

**Pull request merged**